src/test/kotlin/coverosR3z/authentication/AuthenticationBDD.kt

```diff
@@ -53,7 +53,7 @@ class AuthenticationBDD {
         val au = AuthenticationUtilities(authPersistence)
```

Comment:
We should either create an employee and use that ID, or change the name of the scenario from 'employee' to 'user'